test/test-resize-while-forking.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var cluster = require('cluster');
+var master = require('../lib/master');
+var tap = require('tap');
+var workerCount = require('./helpers').liveWorkers;
+
+cluster.setupMaster({
+  exec: 'test/workers/null.js'
+});
+
+tap.test('while workers are forked', function(t) {
+  master.start({size: 10});
+  master.on('startWorker', function() {
+    if (workerCount() === 1) {
+      cluster.fork();
+    }
+    if (workerCount() === 3) {
```

Comment:
Similar to another test.. it almost feels like `workerCount()` is intentionally called multiple times. Are there intended/unintended side effects to doing so?